README.md

```
@@ -348,18 +350,25 @@ Data is organized per billing year under `/users/{userId}/billingYears/{yearId}`
 ## Changelog
 
-### React Migration (2026-03)
+### TipTap Editor, Check Payment Method, QR Codes (2026-04)
+- ✅ TipTap WYSIWYG rich-text editor for invoice templates (TemplateEditor, SubjectEditor)
+- ✅ Custom TipTap nodes: inline %token% pills and block /slash_command tokens
+- ✅ Check payment method type with name, address, phone fields
+- ✅ Bank building icon for Check payment method (branded SVG)
+- ✅ QR code uploads per payment method
+- ✅ 632 React tests + 288 legacy tests + Playwright E2E
+
+### React Migration (2026-03) --- COMPLETE
```

markdownlint-cli2 (0.22.0)

[warning] Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)